src/web/app/components/question-types/question-statistics/question-statistics-calculation/constsum-recipients-question-statistics-calculation.ts

```diff
@@ -54,7 +62,15 @@ export class ConstsumRecipientsQuestionStatisticsCalculation
       const sum: number = answers.reduce((a: number, b: number) => a + b, 0);
       this.totalPointsPerOption[recipient] = sum;
       this.averagePointsPerOption[recipient] = +(answers.length === 0 ? 0 : sum / answers.length).toFixed(2);
+      if (pointsPerOptionToSelf[recipient] === undefined) {
```

Comment:
better to do this like this:
```
this.averagePointsExcludingSelf[recipient] = this.averagePointsPerOption[recipient];
if (this.averagePointsPerOption[recipient] && pointsPerOptionToSelf[recipient] != undefined) {
     this.averagePointsExcludingSelf[recipient] = +(sum - pointsPerOptionToSelf[recipient]) / (answers.length - 1)).toFixed(2);
}
```

Reply:
thanks for the tip! the logic really does flow better this way

just to clarify, what exactly should be changed in `constsum-recipients-question-statistics.component.spec.ts`?